flowmachine/flowmachine/core/query.py

```diff
@@ -227,13 +226,15 @@ def do_get():
                 try:
                     return self._df.copy()
                 except AttributeError:
-                    qur = self.get_query()
+                    qur = f"SELECT {', '.join(self.column_names)} FROM ({self.get_query()}) _"
```

Comment:
Looks like we're doing `', '.join(self.column_names)` a lot, typically inside f-strings. How about adding a convenience property `self.column_names_text` or `self.column_names_str` to wrap this up and make it more readable?